src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -26,6 +26,14 @@
       attach="#filing-history-list"
     />
 
+    <!-- Display for if a court order(s) have been filed -->
+    <v-card v-if="hasCourtOrders" class="my-6 pa-6" elevation="0">
+      <v-icon id="has-court-order-icon">mdi-gavel</v-icon>
+      <span>
+        Court order(s) have been filed on this company. Review the filing history for impacts to business informaiton.
+      </span>
```

Comment:
Spans are supposed to be "inline elements" (not block elements), so the content "should be" on the same line if possible. Sometimes it matters for display, sometimes it doesn't.

One thing, though: in a unit test, the span's text will contain the line breaks here, so you may have to "expect to contain XXX" instead of the more specific "expect to be XXX".

Ideal:
`<span>blah blah blah</span>`

OK:
```
<span>this is the start of this sentence
  and this is the end</span>
```

Reply:
Okay I will change this